Sources/Send/Input/SendInputStateMachine.swift

```diff
@@ -6,11 +6,6 @@ import Combine
 import Foundation
 
 public actor SendInputStateMachine: StateMachine {
-    // Associated types
-    public typealias Action = SendInputAction
-    public typealias Services = SendInputServices
-    public typealias State = SendInputState
-
```

Comment:
Define Action, Services, State types is not needed because the function `accept` has already indicate these types